realcc_tests/test_procedure.py

```diff
@@ -208,8 +208,8 @@ def run(self) -> bool:
     ),
     CattTest(
         "cast first video from youtube playlist on default controller",
-        ["cast", "https://www.youtube.com/watch?v=J4R7M4xH_BA&list=PLhReEC3095X2aQ29-iW2bto0cYPaxch75"],
-        check_data=("status_text", "Casting: The Warrior: A 2015 Psychedelic Trance Mix"),
+        ["cast", "https://www.youtube.com/watch?v=jSL1nXza7pM&list=PLAxEbmfNXWuIhN2ppUdbvXCKwalXYvs8V&index=2&t=0s"],
```

Comment:
Is this going to work with YouTube if we rerun the tests? I wouldn't want our tests to always just fail from now on.

Reply:
Sorry, I meant in the regular test-suite, which makes CI fail. But I can see that I forgot to remove some params from the url. Maybe we'll be lucky after a new commit. :smiley:

Reply:
Just to be clear, the above test in `test_procedure` succeeds.

Reply:
Right, I just marked this as a generic way to leave a comment. I'm not quite clear which one fails, I just don't want a flappy test suite :P